testing/sync-test/src/auth.rs

```diff
@@ -266,12 +268,13 @@ impl TestClient {
         Ok(Self {
             fxa,
             test_acct: acct,
+            autofill_store: AutofillStore::new("./test-autofill-sync.db")?,
```

Comment:
Maybe use ` tempfile::tempfile` here? That might mean you need to store the filename so you can use it below

Reply:
I've tried creating a temp db file with `tempfile::NamedTempFile` ([to get the path](https://docs.rs/tempfile/3.2.0/tempfile/index.html#design)) and ran into the error below which I'm assuming means the file was trashed before the first test object could be added to the database.

```
'add a1: SqlError(SqliteFailure(Error { code: ReadOnly, [extended_code: 1032](https://www.sqlite.org/rescode.html#readonly_dbmoved) }, Some("attempt to write a readonly database")))',
```
So then I tried calling [persist](https://docs.rs/tempfile/3.2.0/tempfile/struct.NamedTempFile.html#method.persist) on the temp file I created with `tempfile::NamedTempFile::new()`, which allows the tests to pass (of course, because it creates a `File` object from the `NamedTempFile`). 

Long story still long, I don't think using a temp file will work here but I'm open to other options.

Reply:
Right, yeah, that's a bit painful. How about we make memory DBs public - something like https://gist.github.com/mhammond/8b70dc0b94f93b3c25be4b723acf0467 ?